roles/maintenance_63_kubernetes_workload/tasks/main.yml

```diff
@@ -0,0 +1,181 @@
+---
+
+- name: This task only serves as a template for the tasks below
```

Comment:
what the ansible?

are we sure there isn't an idiomatic way to do this?

e.g. if it's just about the `when:` bit, why not put it into a block?

Reply:
Nothing in this collection is idiomatic, because it's using Ansible for something that Ansible shouldn't be used for. See https://github.com/adfinis/ansible-collection-maintenance/blob/main/README.md?plain=1#L10-L18

Reply:
oh my